doc/source/cd/methods/cvmdrift.ipynb

```diff
@@ -0,0 +1,136 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[source](../../api/alibi_detect.cd.cvm.rst)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Cramér-von Mises\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "The CVM drift detector is a non-parametric drift detector, which applies feature-wise two-sample [Cramér-von Mises](https://en.wikipedia.org/wiki/Cram%C3%A9r%E2%80%93von_Mises_criterion) (CVM) tests. For two empirical distributions $P(z)$ and $P_{ref}(z)$, the CVM test statistic is defined as\n",
+    "\n",
+    "$$\n",
+    "W = \\sum_{z\\in k} \\left| P(z) - P_{ref}(z) \\right|^2,\n",
```

Comment:
Nitpick but can we use F instead of P? I think P would usually refer to pdfs rather than cdfs which is a little confusing here.

Reply:
Sure thing, will do!